lib/Ethereum/RPC/Client.pm

```diff
@@ -37,7 +37,6 @@ sub AUTOLOAD {
         id      => $self->{id}++,
         method  => $method,
         params  => (ref $_[0] ? $_[0] : [@_]),
-        jsonrpc => "2.0"
```

Comment:
Probably important to include `jsonrpc` if the remote endpoint is truly compliant with the JSON-RPC 2.0 spec.

Reply:
Sure, the Mojo::UserAgent will encode this obj as JSON, but will not include the `jsonrpc` key.

Anyway, it is only important if the remote is fully compliant with JSON-RPC 2.0 (see http://www.jsonrpc.org/specification#request_object.)  Many remote server implementations aren't fully up to the spec though.